testfiles/5ZoneWLHPPlantLoopTower.idf

```diff
@@ -1819,7 +1819,7 @@
     2964,                    !- Lighting Level {W}
     ,                        !- Watts per Zone Floor Area {W/m2}
     ,                        !- Watts per Person {W/person}
-    0.2,                     !- Return Air Fraction
+    ,                        !- Return Air Fraction
```

Comment:
curious of the reason for this change. this would change RA temp and lead to additional iterations.